ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -63,18 +68,39 @@ const (
 // If the --all flag is provided, then list all local ECS task containers.
 // If the --json flag is provided, then output the format as JSON instead.
 func Ps(c *cli.Context) {
+	if err := optionsPreCheck(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
```

Comment:
This log statement is the same as the error text; it would be better to replace it with `logrus.Fatalf(err)`

Reply:
Also- should this be a fatal error? I thought a user can run `ps` without either of these flags.

Reply:
good catch. thx. But probably should be \code logrus.Fatalf(err.Error())

Reply:
> Also- should this be a fatal error? I thought a user can run `ps` without either of these flags.

yes they can. This is to prevent them from using both --file and --task-def flags at the same time

Reply:
> Also- should this be a fatal error? I thought a user can run `ps` without either of these flags.

there will be no error or alert if they do not put any flags. Just show the pwd containers

Reply:
Ah I see. In that case, how about something like this for the error: 
```
fmt.Errorf("%s and %s can not be used together", flags.TaskDefinitionTaskFlag, flags.TaskDefinitionFileFlag)
```

Reply:
Alright, I will fix that